hw3.py

```diff
@@ -0,0 +1,32 @@
+#3.
+tries = 1
+import random
+game = ["student", "homework", "vacation", "notebook", "pencil"]
+print("You have to guess the following word related to school.")
+a = random.choice(game)
+correct = a
+length = len(a)
+length = str(length)
+b = input("The word is " + length + " letters long. Guess a letter: ")
```

Comment:
it should be checked if the value introduce by user has only 1 letter.  Hints can be found [here](https://careerkarma.com/blog/python-isalpha-isnumeric-isalnum/#:~:text=The%20Python%20isalpha()%20method,the%20method%20will%20return%20False%20.)